### ISSUE TITLE
Question regarding image dimension at fine-tuning

### ISSUE THREAD

**Issue by netw0rkf10w** [facebookresearch/SWAG#7]
Hello,

This is great work, congratulations! And thank you for having released the models.

I would like to ask some questions please:

1. Is there a particular reason for choosing `518x518` as crop size for fine-tuning ViT H/14? This value doesn't seem to be very standard to be (I would have expected something like `500x500` or `512x512`). Could you tell me if you did a hyper-parameter search for this value?
2. How much do you expect the accuracy to drop if we use `500x500` instead of `518x518`?

Thank you very much in advance for your response.

**Comment by mannatsingh** [comment]
Hi @netw0rkf10w glad you find the released models useful! Sorry for the delay in response -

> 1. Is there a particular reason for choosing 518x518 as crop size for fine-tuning ViT H/14? This value doesn't seem to be very standard to be (I would have expected something like 500x500 or 512x512). Could you tell me if you did a hyper-parameter search for this value?

The image size is supposed to be divisible by the patch size, and for ViT H/14 our patch size is 14 instead of 16. So the regular `512` size doesn't work and we use the closest number, `518`. This also aligns with the [ViT paper](https://arxiv.org/pdf/2010.11929.pdf) -

> For ImageNet results in Table 2, we fine-tuned at higher resolution: 512 for ViT-L/16 and 518 for ViT-H/14

> 2. How much do you expect the accuracy to drop if we use 500x500 instead of 518x518?

The closest resolution you can use to `500` is `504` (divisible by 14) - I expect the results to be pretty much the same at such a small difference :)

**Comment by netw0rkf10w** [comment]
@mannatsingh Thank you very much for your response. It makes sense totally.
